uk.co.compucorp.civicrm.tasksassignments/settings/tasksandassignments.setting.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+  return [
```

Comment:
I had no idea you could do this, even after reading the [documentation](https://docs.civicrm.org/dev/en/latest/framework/setting/#creating-a-new-setting-in-an-extension) for this I still wasn't clear that this was how you do this :man_shrugging: 

Good to know though